src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,28 @@ title = "Releases"
 weight = 30
 +++
 
+## v0.8.0 Added support for OVN
+
+> This release mainly focused on adding support for interconnecting clusters that use the OVN (ovn-kubernetes) CNI plugin.
+
+Other additions and changes of note:
+
+* The active Gateway now performs periodic health checks on the connections to remote clusters and updates the Gateway
+  connection status and adds latency statistics.
+* Gateways now export the following connection metrics which can be used with Prometheus. These are currently only supported
+  for the Libreswan cable driver.
+  * The count of bytes transmitted and received between Gateways.
+  * The number of connections between Gateways and their corresponding status.
+  * The timestamp of the last successful connection established between Gateways.
+  * The RTT latency between Gateways.
+* The Libreswan cable-driver is now the default.
```

Comment:
```suggestion
* The Libreswan cable driver is now the default. The Strongswan cable driver is deprecated and will be removed in a future release.
```

Should we say this?

Reply:
Not sure - have we said internally that we intend to remove Strongswan? @skitt @nyechiel

Reply:
Yes, we have, and in the upstream calls.

Reply:
Note that it’s “strongSwan” ;-).

Reply:
OK I'll add a separate note item about strongSwan deprecated.